nextflow_schema.json

```
@@ -29,7 +29,7 @@
                     "exists": true,
                     "pattern": "^\\S+\\.yaml$",
                     "description": "Path to a Sopa config file describing the technology used and the pipeline parameters.",
-                    "help_text": "Choose one existing config file amont public one: https://github.com/gustaveroussy/sopa/tree/master/workflow/config",
+                    "help_text": "Choose one existing config file amont public one: https://github.com/gustaveroussy/sopa/tree/main/workflow/config",
```

```suggestion
                    "help_text": "Choose one existing config file among public one: https://github.com/gustaveroussy/sopa/tree/main/workflow/config",
```
just a heads up: this is a very un-nf-core setup.

---

The reason for this choice was for compatibility with the Snakemake version of Sopa, which includes a config. I wanted the nextflow pipeline to be compatible with the existing configs.

How would you update that, knowing that some conf file may be big, e.g. [this one](https://github.com/gustaveroussy/sopa/blob/main/workflow/config/xenium/baysor_prior_small_cells.yaml)?

Also, I made it possible to just choose among [existing online configs](https://github.com/gustaveroussy/sopa/tree/main/workflow/config), limiting this parameter to the URL of the chosen config.

---

Also, as a note, because Nextflow has built-in support for parsing yaml files into the parameters your users will be able to take existing snakemake config files into this pipeline using `nextflow run nf-core/sopa -params-file visium_config.yaml` without any changes.

The only thing that you'll need to do is make sure that the Nextflow params are named the same as the config parameter names.